identity-service/src/notifications/utils.js

```diff
@@ -146,7 +147,32 @@ async function shouldNotifyUser (notificationTarget, prop, tx = null) {
   return { notifyMobile, notifyBrowserPush }
 }
 
+/* We use a JS implementation of the the HashIds protocol (http://hashids.org)
+ * to obfuscate our monotonically increasing int IDs as
+ * strings in our consumable API.
+ *
+ * Discovery provider uses a python implementation of the same protocol
+ * to encode and decode IDs.
+ */
+const HASH_SALT = 'azowernasdfoia'
```

Comment:
jw this salt is identical across IS/DP right?

Reply:
yeah, this code was copied over from the CN actually.

Reply:
Yes, it's the same in CN and DP and now IS

Reply:
@hareeshnagaraj it is

Reply:
decoding + encoding should really live in libs, we've reimplemented this so many times. i'm adding a trello item